src/jit/optimizer.cpp

```diff
@@ -5711,34 +5711,76 @@ bool Compiler::optNarrowTree(GenTree* tree, var_types srct, var_types dstt, Valu
         switch (tree->gtOper)
         {
             case GT_AND:
+                noway_assert(genActualType(tree->gtType) == genActualType(op1->gtType));
                 noway_assert(genActualType(tree->gtType) == genActualType(op2->gtType));
 
-                // Is op2 a small constant than can be narrowed into dstt?
-                // if so the result of the GT_AND will also fit into 'dstt' and can be narrowed
-                if ((op2->gtOper == GT_CNS_INT) && optNarrowTree(op2, srct, dstt, NoVNPair, false))
+                GenTree* opToNarrow;
+                opToNarrow = nullptr;
```

Comment:
`GenTree* opToNarrow = nullptr;` 

same for below 2 variables

Reply:
That results in
```
error C2360: initialization of 'opToNarrow' is skipped by 'case' label [E:\coreclr3\bin
\obj\Windows_NT.x64.Checked\src\jit\standalone\clrjit.vcxproj]
```
To avoid the error I would need to enclose the case in braces. Other switches in this file follow the pattern I have rather than braces over the case so I'll keep it as is.

Reply:
OK then, though the typical way to avoid the error is to add a new scope.